source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard-human-resources.js

```diff
@@ -0,0 +1,104 @@
+﻿$(document).ready(function () {
+    var evaluationStates = ['En curso', 'Esperando Eval Empleado', 'Esperando Eval Responsable', 'Esperando Eval Empresa', 'Lista para devolución', 'Abierta para devolución', 'Finalizada'];
+
+    var ReportDashboard = function (data) {
+        var self = this;
+        this.items = ko.observableArray();
+        if (data) {
+            this.fromJS(data);
+        }
+        this.headers = [
+        { title: 'Periodo', sortable: true, sortPropertyName: 'period', defaultPropertyName: 'fullName', asc: true, activeSort: ko.observable(false) },
+        { title: 'Empleado', sortable: true, sortPropertyName: 'fullName', defaultPropertyName: 'fullName', asc: true, activeSort: ko.observable(false) },
+        { title: 'Puesto', sortable: true, sortPropertyName: 'currentPosition', defaultPropertyName: 'fullName', asc: true, activeSort: ko.observable(false) },
+        { title: 'Seniority', sortable: true, sortPropertyName: 'seniority', defaultPropertyName: 'fullName', asc: true, activeSort: ko.observable(false) },
+        { title: 'Responsable', sortable: true, sortPropertyName: 'responsibleId', defaultPropertyName: 'fullName', asc: true, activeSort: ko.observable(false) },
+        { title: 'Estado', sortable: true, sortPropertyName: 'state', defaultPropertyName: 'fullName', asc: true, activeSort: ko.observable(false), observable: true },
+        { title: '', sortable: false }
+        ];
+        this.sort = commonSort.bind(this);
+        this.defaultSort = function () {
+            this.headers[0].activeSort(false);
+            this.sort(this.headers[0]);
+        }
+        this.isLoading = ko.observable(false);
+    }
+
+    ReportDashboard.prototype.fromJS = function (data) {
+        var self = this;
+        this.items(_.map(data.Evaluations, function (e) {
+            return new Evaluation(e);
+        }));
+    }
+
+    ReportDashboard.prototype.getEvaluationState = function (state) {
+        return this.evaluationStates[state];
+    }
+
+    var Evaluation = function (data) {
+        this.id = '';
+        this.isResponsible = false;
+        this.isEditable = false;
+        this.responsibleId = '';
+        this.fullName = '';
+        this.userName = '';
+        this.period = '';
+        this.currentPosition = '';
+        this.seniority = '';
+        this.state = ko.observable('');
+        this.currentState = '';
+        if (data) {
+            this.fromJs(data);
+        }
```

Comment:
Got it. My original concern turned out to be unnecesary, I thought `fromJs` was returning a new object, and I thought that would be a problem. But that's not the case.